src/pages/HomePage.js

```diff
@@ -34,55 +34,61 @@ const HomePage = () => {
+            <div className='bg-homePagePic bg-cover h-[100vh] bg-center'>
+                <div className='w-full flex justify-center p-4 font-HomePageHeading font-semibold text-black'>
+                    <h1 className='text-5xl p-4 outline outline-2 outline-[#E1E6E6] rounded-lg bg-gray-500 bg-opacity-5'>
                         Make a difference in someone's life
-                    </h1>
+                    </h1>                    
                 </div>
-                <div className='flex justify-center pt-16' data-testid="button-test-id">
+
+                <div className='flex justify-center pt-10' data-testid="button-test-id">
                     <Button title="Create Fundraiser" />
                 </div>
             </div>
-            <div className='p-8 text-5xl font-HomePageHeading font-semibold'>
-                Start a fundraiser
-            </div>
-            {/* STEPS TO FUNDRAISER */}
-            <div className='flex flex-wrap justify-center'>
-                <img src={StepsPic} alt="Logo" className='w-1/2 h-1/2' />
-                <div className='flex flex-col justify-center mx-16'>
-                    
-                    <div className='py-4'>
-                        <div className='text-3xl font-bold'>The Basics</div>
-                        
-                        <div className='text-xl pl-4'>Enter your information</div>
-                    </div>
-                    <div className='py-4'>
-                        <div className='text-3xl font-bold'>What is your cause?</div>
-                        
-                        <div className='text-xl pl-4'>Enter your fundraiser information</div>
-                    </div>
-                    <div className='py-4'>
-                        <div className='text-3xl font-bold'>Support</div>
-                        
-                        <div className='text-xl pl-4'>Enter your payment information</div>
-                    </div>
-                </div>
-
-            </div>
             
             {/* FEATURED FUNDRAISER */}
-            <div className='pt-16 px-8' data-testid="feat-fundraiser-test-id">
+            <div className='px-8 py-4' data-testid="feat-fundraiser-test-id">
                 {/* THe list of objects from the post request is being set as the props */}
                 <HomePageFeatFundraiser data={data}/>
             </div>
+
             {/* CARROUSSEL FUNDRAISER SLIDES */}
-            <div className='pt-16' data-testid="carousel-test-id">
-                <div className='pl-8 text-5xl font-HomePageHeading font-semibold'>
-                    General Fundraisers
+            <div className='px-8 py-4' data-testid="carousel-test-id">
+                <div className='text-5xl font-HomePageHeading font-semibold'>
+                    Active Fundraisers
                 </div>
                 <FundraiserCarousel data={data}/>
             </div>
+
+
+            {/* STEPS TO FUNDRAISER */}
```

Comment:
I think it makes more sense to place this close to the `Create Fundraiser` button as these steps have to do with the fundraiser creation process (granted we don't have that UI done yet)

Reply:
Noted!